packages/react/src/__tests__/ReactChildren-test.js

```diff
@@ -23,6 +23,20 @@ describe('ReactChildren', () => {
     ReactTestUtils = require('react-dom/test-utils');
   });
 
+  const checkReactChildrenFunctionality = (element, callback, context) => {
+    const parentInstance = <div>{element}</div>;
```

Comment:
Let's just copy and paste this part between tests. It's fine for tests to be verbose. That way they're easier to debug.